lib/scheduler.js

```diff
@@ -140,19 +140,19 @@ Scheduler.prototype.startServer = function(successCallback, failureCallback) {
       var host = server.address().address;
       var port = server.address().port;
 
-      console.log('secc-scheduler listening at http://%s:%s', host, port);
+      debug('secc-scheduler listening at http://%s:%s', host, port);
```

Comment:
it's good to use 'console.log' at this time.
'cos, user need to know what's going on